girder/api/v1/group.py

```diff
@@ -247,7 +247,7 @@ def inviteToGroup(self, group, userToInvite, level, quiet, force):
                 })
                 mail_utils.sendEmail(
                     to=userToInvite['email'], text=html,
-                    subject="Girder: You've been invited to a group")
+                    subject="%s: You've been invited to a group" % SettingKey.BRAND_NAME)
```

Comment:
To be clear, this is already Python code, which *is* the server, so there's no HTTP requests made at all.

The `ModelImporter.model('setting')` bit just does a dynamic lookup of the correct Python class to use; this is just causing some global Python variables to be accessed, which is fast.

The `.get()` function call internally runs a PyMongo `find` query to complete, so it will take a bit more time. Eventually, we may want to cache more of these database queries (see @danlamanna 's work in #2274 ), but [premature optimization is the root of all evil](http://wiki.c2.com/?PrematureOptimization), so I wouldn't worry about it right now.